js/models/orderMd.js

```diff
@@ -118,7 +118,7 @@ module.exports = window.Backbone.Model.extend({
 
     response.serverUrl = this.serverUrl;
     response.status = this.status;
-    response.bitcoinValidationRegex = this.bitcoinValidationRegex;
+    response.bitcoinValidationRegex = config.bitcoinValidationRegex;
```

Comment:
This will break the app, the value passed in may be the testnet or mainnet value, this changes it to always be the mainnet value.